envs.py

```diff
@@ -52,8 +52,12 @@ def compute_reward(self):
                isDying(self.obs.players[self.pid]):
                 r -= 1.0
             
+            if not isDying(self.prev_obs.players[opponent_pid]) and \
+               isDying(self.obs.players[opponent_pid]):
+                r += 1.0
+
         #     # We give a reward of -0.01 for every percent taken. The max() ensures that not reward is given when a character dies
-            r += -0.01 * max(0, self.obs.players[self.pid].percent - self.prev_obs.players[self.pid].percent) + 0.01 * (self.obs.players[opponent_pid].percent)
+            r += -0.01 * max(0, self.obs.players[self.pid].percent - self.prev_obs.players[self.pid].percent) + 0.01 * (self.obs.players[opponent_pid].percent - self.prev_obs.players[opponent_pid].percent)
```

Comment:
Oh u fixed it yay. 

qq is it possible that the terms in the second element of the max() are reversed? Shouldn't it be prev percent - current percent + if that's positive then *-0.01? I could be reading it wrong